app/Exceptions/UserProfilePageLookupException.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the GNU Affero General Public License v3.0.
+// See the LICENCE file in the repository root for full licence text.
+
+namespace App\Exceptions;
+
+class UserProfilePageLookupException extends SilencedException
+{
+    private $render;
+
+    public function __construct(callable $render)
```

Comment:
I wonder if renderable exception instead of passing a `callable` would actually be useful 🤔 (or make even less sense 🤷 )

Reply:
maybe if we find another thing needing that...